src/MC/fix_bond_create.cpp

```diff
@@ -120,6 +125,22 @@ FixBondCreate::FixBondCreate(LAMMPS *lmp, int narg, char **arg) :
       itype = force->inumeric(FLERR,arg[iarg+1]);
       if (itype < 0) error->all(FLERR,"Illegal fix bond/create command");
       iarg += 2;
+    } else if (strcmp(arg[iarg],"aconstrain") == 0 &&
+        strcmp(arg[2],"bond/create/angle") == 0) {
```

Comment:
Please replace arg[2] with style, since it has already been parsed
by the parent Fix constructor.